<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_selection</class>
 <widget class="QMainWindow" name="window_selection">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>332</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1141</width>
    <height>332</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1141</width>
    <height>332</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Japanese 練習 - Selection</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_selection_reset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>94</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_hiragana_select_all">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select all
hiragana</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_katakana_select_all">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select all
katakana</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_go_to_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>94</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Go back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
